### PR TITLE
Add CloudFormation to our deployment

### PR DESCRIPTION
**Pull request by frankie297**
Add CloudFormation deployment so it keeps our stacks up to date in version control. This PR also adds a deletion policy to mitigate accidental deletions of tables.

### CODE REVIEW

File: mobile-save-for-later-user-deletion/riff-raff.yaml [hunk 1]
```diff
@@ -9,3 +9,8 @@ deployments:
       functionNames: [mobile-save-for-later-user-deletion-]
       fileName: mobile-save-for-later-user-deletion.jar
       prefixStack: false
+  mobile-save-for-later-user-deletion-cfn:
+    type: cloud-formation
+    app: mobile-save-for-later-user-deletion
+    parameters:
+      templatePath: cfn.yaml
```
Comment (jacobwinch) [inline mobile-save-for-later-user-deletion/riff-raff.yaml:16]: Did you try deploying this to `CODE` yet? I'm not 100% sure but I think we might need to specify the template path as `conf/cfn.yaml`?

File: mobile-save-for-later/conf/cfn.yaml [hunk 2]
```diff
@@ -51,6 +51,7 @@ Mappings:
 Resources:
   SaveForLaterDynamoTable:
     Type: AWS::DynamoDB::Table
+    DeletionPolicy: Retain
```
Comment (jacobwinch) [inline mobile-save-for-later/conf/cfn.yaml:54]: 👍

File: mobile-save-for-later/riff-raff.yaml [hunk 3]
```diff
@@ -9,3 +9,8 @@ deployments:
       functionNames: [mobile-save-for-later-SAVE-, mobile-save-for-later-FETCH-]
       fileName: mobile-save-for-later.jar
       prefixStack: false
+  mobile-save-for-later-cfn:
+    type: cloud-formation
+    app: mobile-save-for-later
+    parameters:
+      templatePath: cfn.yaml
```
Comment (jacobwinch) [inline mobile-save-for-later/riff-raff.yaml:16]: As above!

File: mobile-save-for-later-user-deletion/riff-raff.yaml (hunk 1 above)
Comment (frankie297) [reply mobile-save-for-later-user-deletion/riff-raff.yaml:16]: it worked without adding the `conf` part 🤷‍♀️ shall I add it just incase?

Comment (jacobwinch) [reply mobile-save-for-later-user-deletion/riff-raff.yaml:16]: This part of Riff-Raff has always confused me 🤦 

If it works then there's no need to change it!

**Review by jacobwinch — APPROVED** [review]

**Pull request merged**